src/azure-cli-core/azure/cli/core/local_context.py

```diff
@@ -0,0 +1,101 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import stat
+import shutil
+from six.moves import configparser
+
+from knack.log import get_logger
+from knack.config import _ConfigFile
+from knack.util import ensure_dir, CLIError
+
+STORE = 'store'
+USE = 'use'
+GLOBAL = '__global__'
```

Comment:
Better to leave some comments about what these consts do.

Reply:
added